internal/internal.go

```diff
@@ -842,9 +842,10 @@ func (i *Internal) GetTimeLimitHandler(c echo.Context) error {
 		return echo.NewHTTPError(http.StatusForbidden, "user is not set")
 	}
 
-	if !strings.HasSuffix(user, i.UserSuffix) {
-		user = fmt.Sprintf("%s%s", user, i.UserSuffix)
-	}
+	// user variable is unused, uncomment if it's needed later
+	//if !strings.HasSuffix(user, i.UserSuffix) {
+	//	user = fmt.Sprintf("%s%s", user, i.UserSuffix)
+	//}
```

Comment:
I figured this should stay here in case we need it later, commented out, but the `user` variable is in fact not used after this point.

Reply:
If it's not used, I would go ahead and remove it entirely.